swagger.yaml

```diff
@@ -183,8 +185,12 @@ paths:
           type: string
         - name: source_metadata
           in: query
-          description: '???'
+          description: >-
+            A JSON blob containing extra source specific information.  This
+            field is free-form, but we generally expect the data will follow
+            one of the formats outlined in the `/schema` folder.
```

Comment:
The local URL would just be `/assets/[file name without the folder name]`, e.g. `<a href="/assets/versionista.json">`

You could also use the absolute version, but then staging would point to production’s docs :\

(that would be `https://api.monitoring.envirodatagov.org/assets/versionista.json`)